qwed_infra/guards/iam_guard.py

```
@@ -365,4 +365,22 @@ def to_diagnostic(
             evidence=evidence,
         )
 
+    def to_verification_context(
+        self,
+        result: "InfraDiagnosticResult",
+        formal_statement: str,
+        attestation_token: Optional[str] = None,
+    ) -> "VerificationContextDocument":
```

Ruff (0.16.1)

[error] Undefined name `VerificationContextDocument`

(F821)